push/templates/push/list.html

```diff
@@ -71,6 +71,7 @@
                             <th>{% trans "Application Name" %}</th>
                             <th>VAPID {% trans "Key" %}</th>
                             <th>VAPID {% trans "Key Status" %}</th>
+                            <th></th>
```

Comment:
Clever. I wouldn't have thought so, but this does space the buttons out. Another approach we might want to consider now that we have multiple buttons is adding a new *Management* column for buttons like these. We could still space the *Validate* and *Delete* buttons out using CSS.